NefEditorClient/Catalog/State/Catalog.swift

```diff
@@ -13,7 +13,7 @@ struct Catalog: Equatable {
             FeaturedRecipe(
                 recipe: Recipe(
                     title: "FP with Bow",
-                    description: "Get all modules in Bow 0.8.0 to write FP in Swift",
+                    description: "Get all modules in Bow 0.8.0\nto write FP in Swift",
```

Comment:
I would refrain from breaking lines manually as the card size is dynamic and this can lead to strange text formatting in some cases.
```suggestion
                    description: "Get all modules in Bow 0.8.0 to write FP in Swift",
```